ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml

```diff
@@ -735,6 +759,25 @@ tests:
         requests:
           cpu: "3"
           memory: 600Mi
+- as: e2e-agent
+  optional: true
+  run_if_changed: ^(api/.*|hypershift-operator/controllers/hostedcluster/internal/platform/agent/.*)$
+  steps:
+    cluster_profile: packet-assisted
+    dependencies:
+      HYPERSHIFT_IMAGE: pipeline:hypershift
+    env:
+      ASSISTED_CONFIG: |
+        KUBE_API=yes
+        ENABLE_KUBE_API=true
+        MINIKUBE_DISK_SIZE=70g
+        MINIKUBE_RAM_MB=10240
+        TEST=./src/tests/test_kube_api.py
```

Comment:
does this mean minikube will be the management cluster?

Reply:
Yes.
Assisted-installer test infra is using minikube as the management (hub) cluster.

Reply:
Note that since we are testing the Agent platform we want to management  cluster to run on baremetal.